src/public/index.ts

```diff
@@ -61,6 +61,8 @@ class LKEImageExport {
     this.showCaptions = !this.showCaptions;
     this.ogma.styles.setEdgeTextsVisibility(this.showCaptions);
     this.ogma.styles.setNodeTextsVisibility(this.showCaptions);
+    // @ts-ignore
+    document.getElementById('caption-size')?.disabled = !this.showCaptions;
```

Comment:
what is happening here and why the ts-ignore? just fmi

Reply:
here I am disabling the select size element when the captions are hidden.
the ts-ignore is because TS complaining about disabled attribute doesn't exist in HTMLElement type